Automattic/wp-calypso
# Jetpack: Update Pricing Page with new Design and Pricing

#### Changes proposed in this Pull Request

* Implement design changes from Simplify Pricing Project ( p1HpG7-dcK-p2 ) to the Jetpack Pricing Page

**DO NOT MERGE:** This PR references pricing that is not implemented yet.

**Monthly Selected ( before -> after )**

<img src="https://user-images.githubusercontent.com/2810519/138504415-03784043-7913-476e-9e59-6f4323ad3cdb.png" width=350 align="left">
<img src="https://user-images.githubusercontent.com/2810519/138503569-ceec7eab-0fd7-430a-b5e2-3286798618f2.png" width=350 >
<br />

**Yearly Selected ( before -> after )**
<img src="https://user-images.githubusercontent.com/2810519/138504399-aabd357d-b115-4956-a348-86ccbcecfcef.png" width=350 align="left">
<img src="https://user-images.githubusercontent.com/2810519/138503694-19d16c10-be8d-49e0-aeed-4589cb506e1b.png" width=350 >
<br />

#### Testing instructions


1. Boot branch or use live branch and navigate to `/pricing` with Jetpack Cloud
2. Check that the design matches the above "after" screenshots
